File: python/factorial.py
def factorial(n):
    result = 1
    for i in range(1, n + 1):
        result *= i
    return result

# Input: Enter a number
num = int(input("Enter a number: "))

# Compute and print the factorial
print(f"The factorial of {num} is {factorial(num)}")

File: C++/factorial.cpp
#include <iostream>

unsigned long long factorial(int n) {
    unsigned long long result = 1;
    for (int i = 1; i <= n; i++) {
        result *= i;
    }
    return result;
}

int main() {
    int num;

    // Input: Enter a number
    std::cout << "Enter a number: ";
    std::cin >> num;

    // Compute and print the factorial
    std::cout << "The factorial of " << num << " is " << factorial(num) << std::endl;

    return 0;
}
